### ISSUE TITLE
[Suggestion] Nuclear power unit

### ISSUE THREAD

**Issue by GHXX** [MightyPirates/OpenComputers#2726]
Basic idea is to add a new module (level 1) that can power a computer or microcontroller for an infinite amount of time. The card should not provide power for a whole network. Just one computer and maybe the attached components, like screens and drives.
It should be fairly expensive, maybe a few redstone blocks or enderpearls, or anything you can think of.

Having this would allow everyone to place microcontrollers in the world without needing power.

**Comment by xarses** [comment]
An infinite source isn't needed as there is a config to turn off the power requirement

**Comment by GHXX** [comment]
What if you want some computers to require power and some to not require power? Like some microcontrollers in the world.

**Comment by Techokami** [comment]
There's two options: the cheaty option, and the not-cheaty option.

The cheaty option is to use whatever creative power source is available in the mods that provide power, and place it next to the microcontroller. If you're only using OC, then the Creative Computer Case will act as one.

The not-cheaty option is to set up some sort of passive power generation setup and wire it to the microcontroller. This includes things like solar power mods, or if you have Immersive Engineering you could use a Thermoelectric Generator or a bank of water wheels on a Kinetic Dynamo. And if the microcontroller is out of the way, well then use some cable to run your power connection!

Basically, what your suggestion amounts to is a creative tier version of the battery upgrade. An infinite power source isn't going to be something craftable, and I don't think they can completely overhaul the OC component network to support the "not power the whole network" part.

**Comment by payonel** [comment]
I was letting this ticket sit for a couple days to consider it. With other changes we've made to the power system we've not wanted a free power option. Note that there is now sheep power is the dev builds, will be official in 1.7.2. See carpeted capacitors. It's not free - it has a low chance of killing your sheep/ocelots.

So yeah...I'm not inclined to make free power in a non-creative component

**Comment by GHXX** [comment]
There will probably be a method that checks whether enough power is available or not. In that case it would be as simple as making a new module and inserting a check (if statement) inside the method that checks for the module. It depends on how its coded.

Looks like it just got closed. Well, okay then :P
